addons/binding/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/handler/station/NAStationModuleHandler.java

```diff
@@ -0,0 +1,63 @@
+package org.openhab.binding.netatmo.handler.station;
+
+import static org.openhab.binding.netatmo.NetatmoBindingConstants.*;
+
+import org.eclipse.smarthome.core.library.types.DecimalType;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.eclipse.smarthome.core.types.State;
+import org.openhab.binding.netatmo.handler.NetatmoModuleHandler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.swagger.client.model.NADeviceListBody;
+import io.swagger.client.model.NAModule;
+
+public class NAStationModuleHandler extends NetatmoModuleHandler {
+    private static Logger logger = LoggerFactory.getLogger(NAStationModuleHandler.class);
+
+    public NAStationModuleHandler(Thing thing) {
+        super(thing);
+    }
+
+    @Override
+    protected void updateChannels() {
+        try {
+            NADeviceListBody deviceList = bridgeHandler.getStationApi().devicelist(actualApp, getParentId(), false)
+                    .getBody();
+            for (NAModule module : deviceList.getModules()) {
+                if (module.getId().equalsIgnoreCase(getId())) {
+                    this.module = module;
+                    super.updateChannels();
+                }
+            }
+
+        } catch (Exception e) {
+            logger.error(e.getMessage());
```

Comment:
don't log an error, better provide the error message within the Thing status as this will also be shown in the UI.